### PR TITLE
Add JESP Guide

### PR DESCRIPTION
**Pull request by ivargrimstad**
Signed-off-by: Ivar Grimstad <ivar.grimstad@eclipse-foundation.org>

### CODE REVIEW

**Comment by ivargrimstad** [comment]
The links to the PR templates will work when https://github.com/jakartaee/specifications/pull/335 is merged.

File: content/committees/specification/guide/index.md
Comment (kwsutter) [inline content/committees/specification/guide/index.md]: Wondering if it would be better to replace "projects" with "developers"?  Seems more inclusive.  We need to relate better to the development community.

File: content/committees/specification/guide/index.md
Comment (kwsutter) [inline content/committees/specification/guide/index.md]: Should this be "A Jakarta Specification project must be approved..."?  It's too early in the process to discuss Specifications themselves, right?  We're still talking about the project creation at this point.

File: content/committees/specification/guide/index.md
Comment (kwsutter) [inline content/committees/specification/guide/index.md:36]: The link for the template didn't work for me....  Ahh, never mind...  I see that I need to look at this other PR, https://github.com/jakartaee/specifications/pull/335 for that information.

File: content/committees/specification/guide/index.md
Comment (kwsutter) [inline content/committees/specification/guide/index.md]: Is a Plan review required for only Major versions or all Major and Minor versions?  Maybe this statement could be updated to say ".. in support of a new Specification Major or Minor Version"?

File: content/committees/specification/guide/index.md
Comment (kwsutter) [inline content/committees/specification/guide/index.md]: We should make it clear that a Progress Review is completely optional.  The Project can decide to have one, if they wish to share their progress.  Or, as as you indicated, if nothing has happened in 12 months.  But, we should emphasize somehow that these Progress Reviews are on an "exception basis", they are not part of the normal Spec development process.

File: content/committees/specification/guide/index.md
Comment (kwsutter) [inline content/committees/specification/guide/index.md:27]: Should each of these review modules have a direct link to the specific section of the JESP/EFSP that describes the process in more detail?  I know we have a general link to refer back to the JESP.  But, I'm wondering if more direct links would be helpful.

File: content/committees/specification/guide/index.md
Comment (kwsutter) [inline content/committees/specification/guide/index.md:66]: On each of these sections, we indicate to send an email to the Spec Committee.  But, what does the submitter do after that?  Do they assume success and move on?  No...  In many cases, these reviews require an official ballot, which takes some time (after reviewing).  Is it too restrictive to add a 3rd step to these Reviews that say something like, "3. Wait for a successful xxxx review before proceeding."?

File: content/committees/specification/guide/index.md
Comment (kwsutter) [inline content/committees/specification/guide/index.md]: On this one, the 3rd step would be something like, "3. Wait for the PR approval and merge before proceeding."?

**Review by kwsutter — CHANGES_REQUESTED** [review]
Great stuff, @ivargrimstad!  A few comments to think about.  Always available to discuss further, if necessary.  Thanks!

**Comment by ivargrimstad** [comment]
@kwsutter Thanks for the comments! After quickly browsing through your suggestions, I think all of them will help clarify the message. I will make the updates before I merge tomorrow. Have a nice weekend!

File: content/committees/specification/guide/index.md
Comment (ivargrimstad) [reply content/committees/specification/guide/index.md]: I also added a line specifically saying that plan reviews are not required for service releases

**Pull request merged**
